optuna/samplers/tpe/parzen_estimator.py

```diff
@@ -53,7 +53,7 @@ def _calculate(
             consider_endpoints,  # type: bool
             weights_func  # type: Callable[[int], ndarray]
     ):
-        # type: (...) -> Tuple[List[float], List[float], List[float]]
+        # type: (...) -> Tuple[ndarray, ndarray, ndarray]
```

Comment:
I have a question about the code which uses the result of this method.
It still applies `numpy.asarray` to the returned values as follows:

```python
        s_weights, s_mus, sigmas = ParzenEstimator._calculate(
            mus, low, high, parameters.consider_prior, parameters.prior_weight,
            parameters.consider_magic_clip, parameters.consider_endpoints, parameters.weights)
        self.weights = numpy.asarray(s_weights)
        self.mus = numpy.asarray(s_mus)
        self.sigmas = numpy.asarray(sigmas)
```
Do you think it's necessary? I think they are already `ndarray`.

Reply:
I made a follow-up PR (#600) to handle this issue.